### ISSUE TITLE
Plane Transformation

### ISSUE THREAD

**Issue by arpastrana** [compas-dev/compas#262]
**Describe the bug**
When attempting to transform a `Plane` in place via its `transform()` method, error is raised in line 248 of `compas.geometry._primitives.plane.py` when trying to coerce the first entry into a `float` as it it thinks the output of `compas.geometry.transformations.transform_points()` is a single list . However, the returned value of the latter function is a in fact a nested list.

**To Reproduce**
```
from compas.geometry import Plane
from compas.geometry import Point
from compas.geometry import Vector
from compas.geometry import Rotation

plane = Plane(Point(0, 0, 0), Vector(0, 0, 1))
rot = Rotation.from_quaternion([0, 1, 0, 0])
plane.transform(rot)
```

**Expected behavior**
Transformation successfully applied to Plane.

**Desktop:**
 - OS: Windows 10
 - Python version: 3.7.0
 - Compas version: 0.4.10
 - Python package manager: conda

**Comment by arpastrana** [comment]
Changes proposed in Pull-Request #261

**Comment by arpastrana** [comment]
Closed with #261. Awaiting for merging stuff in the branch feature/geometry_primitives_update, which is soon to merge, in order to address this and a range of other inconsistencies.
